recipes/zospy/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+python:
```

Comment:
This should not be needed.

Reply:
This is related to not using `noarch`, see my response to the previous comment.